pytorch_lightning/plugins/precision/ipu.py

```diff
@@ -30,6 +30,8 @@ class IPUPrecisionPlugin(PrecisionPlugin):
     """Precision plugin for IPU integration."""
 
     def __init__(self, precision: int) -> None:
+        if precision not in (16, 32):
+            raise MisconfigurationException(f"`Trainer(accelerator='ipu', precision={precision!r})` is not supported.")
```

Comment:
The error message should include what *is supported* so users can quickly resolve the issue without digging into the source code or docs.
```suggestion
        supported_precision_values = (16, 32)
        if precision not in supported_precision_values:
            raise MisconfigurationException(f"`Trainer(accelerator='ipu', precision={precision!r})` is not supported. `precision` must be one of: {supported_precision_values}")
```